cmake/alpakaCommon.cmake

```
@@ -179,6 +179,10 @@ if(CMAKE_CXX_COMPILER_ID MATCHES "Clang" AND CMAKE_CXX_COMPILER_VERSION VERSION_
     message(FATAL_ERROR "Clang versions < 4.0 are not supported!")
 endif()
 
+if(CMAKE_CXX_COMPILER_ID MATCHES "Intel ") # match Intel xxxx.x.x but not IntelLLVM xxxx.x.x
+    message(FATAL_ERROR "Support for the Intel Classic compiler (icpc) has stopped. Please upgrade to the Intel LLVM compiler (ipcx)!")
```

```suggestion
    message(FATAL_ERROR "The Intel Classic compiler (icpc) is no longer supported. Please upgrade to the Intel LLVM compiler (ipcx)!")
```

Also, shouldn't we make this a warning instead? If a user wants to use `icpc` anyway they can try - we shouldn't block them from doing so.

---

ok!